parser/Parser.go

```diff
@@ -0,0 +1,56 @@
+package parser
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/anxiousmodernman/dirz/lexer"
+	"github.com/anxiousmodernman/dirz/model"
+	"github.com/anxiousmodernman/dirz/token"
+)
+
+func isEOF(theToken token.Token) bool {
+	return theToken.Type == token.TOKEN_EOF
+}
+
+func Parse(fileName string) model.DirzContext {
```

Comment:
@swizzard I imagine this will return a parsed tree type thing to the caller, which is currently the main function